presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -102,8 +102,66 @@ Property Name                                      Description
 ``hive.immutable-partitions``                      Can new data be inserted into existing partitions?           ``false``
 
 ``hive.max-partitions-per-writers``                Maximum number of partitions per writer.                     100
+================================================== ============================================================ ==========
+
+Amazon S3 Configuration
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The Hive connector also allows querying data stored in Amazon S3. The SQL support 
+for S3 tables is the same as for HDFS tables.
+
+To access tables stored in S3, you must specify the AWS credential properties 
```

Comment:
I think the beginning of the docs mentions it that the configuration has to go to .properties file. So didnt add that here since it seemed redundant